packages/itinerary-body/src/styled.tsx

```diff
@@ -237,12 +237,6 @@ export const Destination = styled.div`
   text-align: center;
 `;
 
-export const DetailsColumn = styled(LightBorderDiv)`
```

Comment:
Potentially a breaking change (a externally visible export is removed).

Reply:
Is there a way to check if this is a breaking change? It looks like all the instances of `DetailsColumn` have been removed